doc/proposals/ansible-operator-webhooks.md

```diff
@@ -0,0 +1,81 @@
+# Ansible Operator Webhooks
```

Comment:
WDYT about using the [template](https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/TEMPLATE.md)?

Reply:
I hadn't seen that before, it seems pretty heavy though, is there a lighter alternative or do we need to fill out the whole doc?

Reply:
We can remove the sections which no make sense for the proposal, but I think we should follow up it.

Reply:
Solved